rplugin/python3/deoplete/sources/tabnine.py

```diff
@@ -144,8 +145,15 @@ def _request(self, name, **params):
             self._restart()
             return
 
-        output = proc.stdout.readline()
         try:
+            # try to read from proc's stdout
+            events = selector.select(timeout=0.5)
```

Comment:
We advise for 1 second timeout, as we've also implemented in vscode & intellij plugins:
https://github.com/codota/tabnine-intellij/blob/b8ddb130ec007a757e6fc181a9bc745cf946bac7/src/main/java/com/tabnine/binary/BinaryRequestFacade.java#L26

Reply:
@Shougo reported better behaviour with 0.5 seconds. Was also my experince.

Reply:
`0.5` is for block timeout.  It is not block timeout.  So I think to use `1.0` is acceptable.